src/components/EvModuleList.vue

```diff
@@ -25,17 +25,41 @@
     </v-expansion-panel>
     <v-expansion-panel :disabled="config_list.length == 0">
       <v-expansion-panel-header>
-        {{ config_list.length == 0 ? "No configs available" : "Available configs" }}</v-expansion-panel-header
-      >
+        {{ config_list.length == 0 ? "No configs available" : "Available configs" }}
+      </v-expansion-panel-header>
       <v-expansion-panel-content>
         <v-sheet class="d-flex align-center my-1 pa-2" elevation="1" v-for="config in config_list" :key="config">
           {{ config }}
           <v-spacer />
-          <v-btn icon @click="load_config(config)">
+          <v-btn icon @click="check_if_config_can_be_loaded(config)">
             <v-icon>mdi-upload</v-icon>
           </v-btn>
         </v-sheet>
       </v-expansion-panel-content>
+      <v-dialog
+          v-model="dialog"
```

Comment:
A better variable name would probably be something like "showDialog". Additionally it would make sense to make it more specific in the case of multiple dialogs.